qa/rpc-tests/invalidblockrequest.py

```diff
@@ -25,9 +25,12 @@
 # Use the ComparisonTestFramework with 1 node: only use --testbinary.
 class InvalidBlockRequestTest(ComparisonTestFramework):
 
+
     ''' Can either run this test as 1 node with expected answers, or two and compare them. 
         Change the "outcome" variable from each TestInstance object to only do the comparison. '''
     def __init__(self):
+        #This test requires mocktime
+        enable_mocktime()
```

Comment:
This test also doesn't seem to do anything with transactions and works fine for me locally without any changes.